### PR TITLE
Add license and contributing files

### PR DESCRIPTION
**Pull request by gwarf**
Add license, authors and contributing files.

### CODE REVIEW

**Comment by gwarf** [comment]
@maarten-litmaath could you please kindly review this, especially the `AUTHORS` file?

**Comment by maarten-litmaath** [comment]
Bonjour Baptiste,
the contents of the AUTHORS file are incorrect.  Also see my next comment.
The contents of the file should be as follows:

Maintainers
-----------
Baptiste Grenier <baptiste.grenier@egi.eu>

Original Authors
----------------
Andrea Sciaba <Andrea.Sciaba@cern.ch>

Contributors
------------
Laurence Field <Laurence.Field@cern.ch>
Maarten Litmaath <Maarten.Litmaath@cern.ch>
Maria Alandes Pradillo <maria.alandes.pradillo@cern.ch>

**Comment by maarten-litmaath** [comment]
Hi again,
upstream is 2 bugfix versions ahead:

http://linuxsoft.cern.ch/wlcg/centos7/x86_64/  -->

lcg-info-1.12.3-1.noarch.rpm 
lcg-info-1.12.4-1.el7.noarch.rpm

**Comment by gwarf** [comment]
> Hi again,
> upstream is 2 bugfix versions ahead:
> 
> http://linuxsoft.cern.ch/wlcg/centos7/x86_64/ -->
> 
> lcg-info-1.12.3-1.noarch.rpm
> lcg-info-1.12.4-1.el7.noarch.rpm

In fact this are also the versions that are in UMD: http://repository.egi.eu/sw/production/umd/4/centos7/x86_64/updates/

The fact is that this GitHub repository is expected to be the new home and so the upstream, it was moved there as part of the original migration of BDII components from CERN to EGI...
Are you still maintaining it somewhere in an accessible repo?
We can also look at the RPM content to see the changes on the perl file and man page but would be cleaner to be able to check the repo if any.

I pushed an update for the AUTHORS, thanks for being our Indiana Jones with those components :)

**Comment by maarten-litmaath** [comment]
Hi again,
I can look into what changed and commit the fixes here.
I had applied them ad-hoc, there is no other source repository.
How urgent is this business?

**Comment by gwarf** [comment]
So we would like to merge #2 and release a new version that would address #1 that was opened quite some time ago in GGUS: https://ggus.eu/index.php?mode=ticket_info&ticket_id=140478.
Would be good if we could finalise this task soon. We can quite easily make a diff of the files too if you don't have much time now for making a proper PR.

**Comment by maarten-litmaath** [comment]
Hi again,
if you could have a quick looks at the diffs, that would be great, merci!
IIRC they were fairly trivial...

**Comment by gwarf** [comment]
I ingested a diff from the last version from UMD (seems the same as from the CERN repo) and finally it builds!
So it's ready for your final validation and merge, thanks!
I will send tag a new version 1.12.4 and then we will merge #2 and release 1.12.5 to be submitted to UMD.

**Comment by maarten-litmaath** [comment]
Salut,
what would you want me to validate and merge still?

File: lcg-info.spec [hunk 1]
```diff
@@ -11,9 +11,10 @@ BuildRequires: make
 BuildRequires: rsync
 BuildRequires: gcc
 Requires: perl-LDAP
+URL: https://github.com/EGI-Foundation/lcg-ino
```
Comment (andrea-manzi) [inline lcg-info.spec]:
```suggestion
URL: https://github.com/EGI-Foundation/lcg-info
```

**Comment by gwarf** [comment]
No specific need @maarten-litmaath, if you are fine with the change EGI colleagues can make the formal review approving it so that we can merge it.

**Review by andrea-manzi — APPROVED** [review]
thanks a lot, really good!

**Pull request merged**